.gitignore

```diff
@@ -1,5 +1,24 @@
+# Composer Dependencies
 /vendor/*
+/bin
```

Comment:
A fresh Composer install of Cake3 using Composer does not move or create any bin scripts.

I'm messing with this.